packages/language-service/test/project/app/parsing-cases.ts

```diff
@@ -141,9 +145,7 @@ export class TestComponent {
   @Output('test') testEvent = new EventEmitter();
 } /*EndTestComponent*/
 
-@Component({
-  templateUrl: 'test.ng',
-})
+@Component({templateUrl: 'test.ng', animations: [trigger('openClose')]})
```

Comment:
```suggestion
@Component({templateUrl: 'test.ng', animations: [trigger('openClose')],})
```

nit: just to make easier to read ))